### PR TITLE
Fixing quotes of multiple filenames in Galaxy

### PR DESCRIPTION
**Pull request by bgruening**
Ones and for all, I added a text file with a space in the name ;)

### CODE REVIEW

File: galaxy/wrapper/multiBamSummary.xml [hunk 1]
```diff
@@ -16,8 +16,8 @@
             @THREADS@
 
             --outFileName '$outFile'
-            --bamfiles '#echo "' '".join($files)#'
-            --labels '#echo "' '".join($labels)#'
+            --bamfiles #echo " ".join($files)#
+            --labels #echo " ".join($labels)#
```
Comment (dpryan79) [inline galaxy/wrapper/multiBamSummary.xml]: I assume we no longer need the `#` at the end of each of these lines.

Comment (bgruening) [inline galaxy/wrapper/multiBamSummary.xml]: It more or less closes the cheetah template. It could be removed yes.

**Comment by dpryan79** [comment]
lgtm!

**Comment by bgruening** [comment]
Thanks Devon!

**Pull request merged**